experimental/LieAlgebras/src/RootSystem.jl

```diff
@@ -438,13 +450,13 @@ function Base.:(*)(q::RationalUnion, r::RootSpaceElem)
 end
 
 function Base.:(+)(r::RootSpaceElem, r2::RootSpaceElem)
-  @req root_system(r) === root_system(r2) "$r and $r2 must belong to the same root space"
+  @req root_system(r) === root_system(r2) "parent root system mismatch"
```

Comment:
these strings get interpolated even in the case that no error is thrown. Withouth string interpolation, there is basically no overhead